boards/riscv/longan_nano/Kconfig.defconfig

```diff
@@ -11,4 +11,14 @@ config BOARD
 config DISK_DRIVER_SDMMC
 	default y if DISK_DRIVERS
 
+if DISPLAY
+
+config SPI
+	default y
+
+config ST7735R
+	default y
+
+endif # DISPLAY
+
```

Comment:
None of this should be necessary, https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/display/Kconfig.st7735r#L9

Reply:
Thank you for pointing out.
Removed it.